exam-01/section-2-3.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
```

Comment:
Section 2: `15/15`
Section 3: `8/20`

<img width="568" alt="Screenshot 2024-07-30 at 10 44 07 PM" src="https://github.com/user-attachments/assets/3818fbd9-5cf2-48b7-91a4-3bb277587e4d">


- Good work trying to pull everything together.
- The CSS is still very lacking as the UI looks all over the places
- Bootstrap is being loaded but we do not use it at all